css/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0 auto;
+  padding-left: 5px;
+}
+.firstfield {
```

Comment:
Use kebab-case (`.first-field` is better)

Reply:
Edited.